trippo/src/components/navBar/Navbar.tsx

```diff
@@ -68,11 +81,32 @@ const Navbar = (props: { history: any }) => {
               <img alt="Trippo Logo" src="trippo.png" width="120"></img>
             </sc.LogoButton>
           </sc.Logo>
+          <IconButton
+            style={{ position: "absolute", right: 20 }}
+            onClick={handleDropdownClick}
+          >
+            <AccountCircle style={{ width: 35, height: 35, color: GREY }} />
```

Comment:
```suggestion
            <AccountCircle style={{ width: 35, height: 35, color: BLACK}} />
```
just to keep it consistent with the other ones :)